### PR TITLE
OEL-3024: Added tests for new class on pagination.

### PR DESCRIPTION
**Pull request by tibi2303**
Jira issue(s):
- [D8AGE-???](https://webgate.ec.europa.eu/CITnet/jira/browse/D8AGE-???)

### CODE REVIEW

File: tests/src/PatternAssertion/PaginationPatternAssert.php [hunk 1]
```diff
@@ -25,6 +25,10 @@ protected function getAssertions(string $variant): array {
       'size' => [
         [$this, 'assertSize'],
       ],
+      'list_attributes' => [
+        [$this, 'assertElementAttributes'],
+        '.pagination.js-pager__items',
```
Comment (drishu) [inline tests/src/PatternAssertion/PaginationPatternAssert.php]: hardcoding this is wrong, we set what is expected, we need to assert against that

**Review by drishu — CHANGES_REQUESTED** [review]

File: tests/src/PatternAssertion/PaginationPatternAssert.php (hunk 1 above)
Comment (tibi2303) [reply tests/src/PatternAssertion/PaginationPatternAssert.php]: changed the selector.

**Review by drishu — DISMISSED** [review]

File: tests/src/Kernel/fixtures/markup_rendering_patterns/pagination.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ pagination:
       settings:
         size: 'sm'
         alignment: 'center'
+        list_attributes: 'class="list-class" id="list-id" data-list-attribute="list-attr-value"'
```
Comment (brummbar) [inline tests/src/Kernel/fixtures/markup_rendering_patterns/pagination.yml]: As I said in previous reviews, do not change the default test. In here we should just test the default values.
Add a new one, with the extra attributes.

**Review by brummbar — CHANGES_REQUESTED** [review]

File: tests/src/Kernel/fixtures/markup_rendering_patterns/pagination.yml (hunk 2 above)
Comment (tibi2303) [reply tests/src/Kernel/fixtures/markup_rendering_patterns/pagination.yml]: did it in a separate test.

**Review by drishu — APPROVED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**